src/notification.scss

```diff
@@ -29,314 +31,278 @@ $block: #{$fd-namespace}-notification;
   $fd-notification-indicator-border-color-information: var(--sapInformationBorderColor) !default;
 
-  $fd-notification-title-color: var(--sapGroup_TitleTextColor) !default;
-  $fd-notification-header-border-color: var(--sapPageHeader_BorderColor) !default;
-  $fd-notification-content-text-color: var(--sapContent_LabelColor) !default;
+  $fd-notification-padding-left: 0.5rem;
+  $fd-notification-padding-right: 1rem;
+  $fd-notification-content-offset: 0.75rem;
+  $fd-notification-padding-mobile: 0.75rem;
+  $fd-notification-reset-offset: 0;
 
   // mixins
+
   @mixin fd-priority-indicator($type) {
-    margin: 1px 0 0 0;
-    padding: 1rem 0.5rem 0 1rem;
+    @include fd-icon("message-#{$type}");
+  }
 
-    @include fd-rtl() {
-      padding: 1rem 1rem 0 0.5rem;
+  @mixin fd-hide-on-mobile {
+    @include fd-screen(s) {
+      display: none !important;
     }
 
-    @include fd-icon("message-#{$type}");
+    @include fd-screen(m) {
+      display: inline-block !important;
+    }
   }
 
-  // ------------------------ CONTAINER ------------------------
   @include fd-reset();
 
-  max-width: $fd-notification-max-width;
   background-color: $fd-notification-background-color;
   box-shadow: $fd-notification-box-shadow;
   border-radius: $fd-notification-border-radius;
-  border-left: 0.375rem solid;
-  border-left-color: $fd-notification-indicator-border-color-default;
-  margin-bottom: 0.5rem;
+  margin-bottom: 1.25rem;
+  width: 100%;
+  max-width: $fd-notification-max-width-m;
+  box-sizing: border-box;
+  padding: 1rem 0.5rem 1rem 1rem;
 
   @include fd-rtl() {
-    border-left: none;
-    border-right: 0.375rem solid;
-    border-right-color: $fd-notification-indicator-border-color-default;
+    padding-right: $fd-notification-padding-right;
+    padding-left: $fd-notification-padding-left;
   }
 
-  // ------------------------ HEADER ------------------------
-  &__header {
+  &__body {
     @include fd-reset();
 
     display: flex;
-    box-shadow: inset 0 -0.0625rem $fd-notification-header-border-color;
-  }
-
-  &__indicator {
-    @include fd-reset();
-
-    &--error {
-      @include fd-priority-indicator('error');
-
-      color: $fd-notification-indicator-color-error;
-    }
-
-    &--success {
-      @include fd-priority-indicator('success');
-
-      color: $fd-notification-indicator-color-success;
-    }
-
-    &--warning {
-      @include fd-priority-indicator('warning');
-
-      color: $fd-notification-indicator-color-warning;
-    }
-
-    &--information {
-      @include fd-priority-indicator('information');
+    justify-content: space-between;
 
-      color: $fd-notification-indicator-color-information;
+    &--message {
+      padding-top: 1rem;
     }
   }
 
-  &__title {
+  &__content {
     @include fd-reset();
 
-    color: $fd-notification-title-color;
-    padding: 1rem 0;
-    margin-right: 1rem;
-    font-weight: 700;
     width: 100%;
+    margin-left: $fd-notification-content-offset;
```

Comment:
![image](https://user-images.githubusercontent.com/26483208/93488019-2d19b180-f906-11ea-8548-63407ab52ef4.png)
This should be `1rem` and `0.75rem` only on mobiles